ui/app/AppLayouts/Chat/ChatLayout.qml

```diff
@@ -134,4 +187,37 @@ StackLayout {
             }
         }
     }
+
+    Component {
+        id: communityIntroDialogPopup
+        CommunityIntroDialog {
+            id: communityIntroDialog
+
+            property string communityId
+
+            onJoined: {
+                root.rootStore.requestToJoinCommunityWithAuthentication(communityIntroDialog.communityId, root.rootStore.userProfileInst.name)
+            }
+
+            onCancelMembershipRequest: {
+                root.rootStore.cancelPendingRequest(communityIntroDialog.communityId)
+                mainViewLoader.item.isInvitationPending = root.rootStore.isCommunityRequestPending(communityIntroDialog.communityId)
```

Comment:
Breaking the binding

Reply:
Same as my comment here: https://github.com/status-im/status-desktop/pull/11134/files#r1233958898